plugins/modules/ndo_port_channel_interface.py

```diff
@@ -0,0 +1,382 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright: (c) 2024, Gaspard Micol (@gmicol) <gmicol@cisco.com>
+
+# GNU General Public License v3.0+ (see LICENSE or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+
+__metaclass__ = type
+
+ANSIBLE_METADATA = {
+    "metadata_version": "1.1",
+    "status": ["preview"],
+    "supported_by": "community",
+}
+
+DOCUMENTATION = r"""
+---
+module: ndo_port_channel_interface
+short_description: Manage Port Channel Interfaces on Cisco Nexus Dashboard Orchestrator (NDO).
+description:
+- Manage Port Channel Interfaces on Cisco Nexus Dashboard Orchestrator (NDO).
+- This module is only supported on ND v3.2 (NDO v4.4) and later.
```

Comment:
Are you sure its NDO v4.4? I can see the changes for `interfaceDescriptions` in the v4.3 API changelog.
https://developer.cisco.com/docs/nexus-dashboard-orchestrator/4-3-1/api-changelog/#release-431-2024-03-01

Reply:
you're right it was introduced in 4.3 but the patch method for Fabric Resource Template was introduced in 4.4. This a problem I had before creating the module and talked about with the engineering team. We built most of our modules for template objects with patch method for convenience. Therefore this module in particular can only be used for 4.4 and later.

Reply:
I see, okay makes sense. :)